packages/lib/src/vis/shared/CameraProvider.tsx

```diff
@@ -0,0 +1,40 @@
+import { useThree } from '@react-three/fiber';
+import type { ReactNode } from 'react';
+import { createContext, useContext } from 'react';
+import { Vector2, Vector3 } from 'three';
+
+import { useFrameRendering } from '../hooks';
+import { useAxisSystemContext } from './AxisSystemContext';
+
+export interface CameraContextValue {
+  worldToHtml: (point: Vector2 | Vector3) => Vector2;
+}
+
+const CameraContext = createContext({} as CameraContextValue);
```

Comment:
I've put the context in the same file for simplicity. I'll do the same with `AxisSystemProvider` so that it's easier to get to the provider from a consumer component in VSCode.

Reply:
THANK YOU